flow/scripts/open.tcl

```diff
@@ -60,12 +60,43 @@ proc read_timing {input_file} {
 if {[env_var_equals GUI_TIMING 1]} {
   puts "GUI_TIMING=1 reading timing, takes a little while for large designs..."
   read_timing $input_file
+
+  puts "Prerendering..."
+  gui::save_display_controls
+
+  set height [[[ord::get_db_block] getBBox] getDY]
+  set height [ord::dbu_to_microns $height]
+  set resolution [expr $height / 1000]
+
+  foreach clock [get_clocks *] {
+    if { [llength [get_property $clock sources]] > 0 } {
+      set clock_name [get_name $clock]
+      save_clocktree_image -clock $clock_name \
+          -width 1024 -height 1024 \
+          $::env(REPORTS_DIR)/cts_$clock_name.webp
+      gui::select_clockviewer_clock $clock_name
+      save_image -resolution $resolution $::env(REPORTS_DIR)/cts_${clock_name}_layout.webp
+    }
+  }
```

Comment:
@maliberty How do I unselect the clock tree and return the GUI to the default view?

`make gui_cts`

![image](https://github.com/user-attachments/assets/206cd8f8-ad81-4235-8bb2-c7f62a0675b5)